templates/shared/_ubuntu_pro_on_public_cloud.html

```diff
@@ -10,18 +10,18 @@ <h2>Ubuntu Pro on public cloud</h2>
       </ul>
       <div class="row">
         <div class="col-4">
-          <p>
-            <a class="p-link--external" href="https://assets.ubuntu.com/v1/feafb61c-Ubuntu_Pro_AWS_04.11.19.pdf">Get the Ubuntu Pro for AWS datasheet</a>
-          </p>
           <a class="p-button--positive" href="/aws/pro">Learn about Ubuntu Pro for AWS</a>
         </div>
         <div class="col-4">
-          <p>
-            <a class="p-link--external" href="https://assets.ubuntu.com/v1/27a386bc-Ubuntu_Pro_Azure_03.04.20.pdf">Get the Ubuntu Pro for Azure datasheet</a>
-          </p>
+
           <a class="p-button--positive" href="/azure/pro">Learn about Ubuntu Pro for Azure</a>
         </div>
       </div>
+      <div class="row">
+        <div class="col-4">
+            <a class="p-button--positive" href="/gcp/pro">Learn more about Ubuntu Pro for GCP</a>
```

Comment:
As the other 2 buttons say "Learn about" maybe we should remove the "more"? Just as the button looks odd being a lot wider than the one above it.